target/linux/ath79/image/generic-ubnt.mk

```diff
@@ -24,6 +24,20 @@ define Build/mkubntimage-split
 	rm $@.old1 $@.old2 )
 endef
 
+# similar to the UBNT XM devices the newer UBNT WA devices need a split image. But it expects
+# a higher firmware version or else it won't update
+define Build/mkubntimage-split-wa
```

Comment:
This one is indeed a near to identical copy of the Build/mkubntimage-split. Do the older XM devices have an issue with a higher version number than v6.0.0? If so, we should pass the version number (similar to UBNT_TYPE) from the image build code instead of duplicating the whole image build code just for this purpose.

Reply:
Yeah you are right. It was always a hack and I just forgot about it. I'll fix that